filesystems/oss/src/main/scala/cromwell/filesystems/oss/nio/OssStorageFileSystemProvider.scala

```diff
@@ -267,7 +275,11 @@ final case class OssStorageFileSystemProvider(config: OssStorageConfiguration) e
     }
 
     val exists = OssStorageRetry.from(
-      () => ossClient.doesObjectExist(ossPath.bucket, ossPath.key)
+      () => {
+        val request = new GenericRequest(ossPath.bucket, ossPath.key)
+        request.setLogEnabled(false)
```

Comment:
seems like there should be a thing that takes an ossPath and returns a request with logging disabled

Reply:
Sure. Someday. https://github.com/broadinstitute/cromwell/issues/3526